### PR TITLE
fix(uniqueid): add unique id helper to handlebars

### PR DESCRIPTION
**Pull request by dgutride**

### CODE REVIEW

File: src/site/pages/contribution.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ There are several properties available on the Example component to help with sty
 <Example heading="Heading" minHeight="20em" docs={docReference} fullPageOnly="">{example}</Example>;
 ```
 
+If an ID is needed for an element, please use the Handlebars unique ID helper.  This can be used by appending {{uniqueId} to the ID attribute so it looks like this:  id="badge-{{uniqueId}}".
```
Comment (srambach) [inline src/site/pages/contribution.md]: Can you backtick the code bits so they look like code? 
Also, `{{uniqueID}` needs one more `}`
Thanks for remembering to add it to the docs!

**Comment by jgiardino** [comment]
I'm assuming that each instance of `{{uniqueId}}` unique, and that there's no way to reference this value within another attribute, like `for` or `aria-labelledby`?

**Comment by dgutride** [comment]
@jgiardino  - it is random each time, but a dev could generate it in the example and pass it into the handlebars file (which could just prefix it over and over again with different prefixes in order to build out a full component and use the for/aria tags).

**Comment by jgiardino** [comment]
Sounds good! Thanks, @dgutride

**Comment by srambach** [comment]
Thanks, @dgutride !

**Pull request merged**